src/base.jl

```diff
@@ -31,13 +33,27 @@ const Components = Dict{DataType, Dict{String, <:Component}}
     # Keyword arguments
 
     * `runchecks`::Bool : run available checks on input fields
```

Comment:
Can you add documentation on what will happen if runchecks is true and an error is found?